plugins/notifiers/slack/client.go

```diff
@@ -67,11 +76,16 @@ func (n *notifier) Notify(items []domain.Notification) []error {
 	return errs
 }
 
-func (n *notifier) sendMessage(channel, text string) error {
+func (n *notifier) sendMessage(channel, messageBlock string) error {
 	url := slackHost + "/api/chat.postMessage"
-	data, err := json.Marshal(map[string]string{
+	var messageblockList []interface{}
+
+	if err := json.Unmarshal([]byte(messageBlock), &messageblockList); err != nil {
+		return fmt.Errorf("error in parsing message block %s", err)
```

Comment:
if the json unmarshal returns an error let's consider it as a normal string and send to slack using `text` field instead of `blocks`. This also keeps this change backward compatible

Reply:
Isn't it good to know the error than sending it as text.?
As we would be aware that there is something wrong in configuring notification.

We already has added the documentation for this change.